react/features/filmstrip/components/web/Filmstrip.js

```diff
@@ -108,7 +108,9 @@ class Filmstrip extends Component<*> {
         // will get updated without replacing the DOM. If the known DOM gets
         // modified, then the views will get blown away.
         const reduceHeight
```

Comment:
Am I right that the variables interfaceConfig and _toolboxVisible are only used in the class to calculate reducedHeight? If yes, maybe worth considering turning reducedHeight into a prop calculated in mapStateToProps and reduce the number of times the render is invoked.